test/System.IdentityModel.Tokens.Jwt.Tests/JwtPayloadTest.cs

```diff
@@ -58,9 +58,13 @@ public void GetSets()
             JwtPayload jwtPayload = new JwtPayload();
             Type type = typeof(JwtPayload);
             PropertyInfo[] properties = type.GetProperties();
+#if NET9_0_OR_GREATER
```

Comment:
This is a temporary fix to complete this work. Need to look in-depth into fixing this test properly.